### DISCUSSION TITLE
Cannot see waveform for mp3 file

### DISCUSSION THREAD

**Discussion by gitty0101** [mifi/lossless-cut#1930] [Q&A]
New user. I have a 5 hour long audiobook mp3 file, and I have been unable to view the waveform. I wish to see the troughs, which will hopefully be the silence between chapters.

The audio is a simple bar. I see the notification 'Zoom in to view waveform'. I have tried zoom from 1X to 4096X and neither view shows an actual wave.

It is my understanding that a micro waveform should be visible where the bar currently is, and a larger waveform can be made to show where the video would normally display if this was a video file??

My larger waveform is simply a red vertical line. I can actually drag where this line is but cannot see a waveform.

Any help appreciated.

**Comment by mifi** [comment]
does this happen with all audio files, or do you have a sample file that reproduces the problem? if you open  the javascript console log (Menu: Tools > Toggle Developer Tools), do you see any error?
